### PR TITLE
Added UserSignUp event

### PR DESCRIPTION
**Pull request by whatevercamps**

### CODE REVIEW

**Comment by joaolcorreia** [comment]
Schema is valid but permissions to storageloadersnowcat were removed so I cannot approve this until storageloadersnowcat is owner of the snowplow schema.

```
ALTER SCHEMA snowplow OWNER TO storageloader;
```

image

**Comment by joaolcorreia** [comment]
- Schema is valid
- Table is created in Redshift
- Schemas are in place ready to be used

**Comment by joaolcorreia** [comment]
@travs15 @rafaelfrieri1 please approve and merge.

File: schemas/com.torrelabs/UserSignUp/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        }
+    },
+    "additionalProperties": true,
+    "type": "object",
+    "required": [
+        "email",
+        "name",
+        "subjectIdentifier"
+    ]
+}
```
Comment (travs15) [inline schemas/com.torrelabs/UserSignUp/jsonschema/1-0-0:58]: Please add this new line to avoid the warning message from github
```suggestion
}

```

**Review by travs15 — APPROVED** [review]
Hi @whatevercamps LGTM, I just left a comment inside the review

**Comment by joaolcorreia** [comment]
Hey guys please DO NOT make any changes to schemas after they are deployed. Schemas as immutable, I cannot change them.

**Pull request merged**